src/main/java/Main.java

```diff
@@ -29,8 +29,23 @@ public static void main(String[] args) {
                     duke.markDone(index);
                     break;
                 }
-                duke.addList(line);
-                break;
+                if(line.contains("todo")){
+                    line = line.substring(5);
+                    duke.addList(line,"T");
+                    break;
+                }
+                if(line.contains("deadline")){
+                    String time = line.substring(line.indexOf("/")+1);
+                    line = line.substring(9,line.indexOf("/")) + "(" + time + ")";
+                    duke.addList(line,"D");
+                    break;
+                }
+                if(line.contains("event")){
+                    String time = line.substring(line.indexOf("/")+1);
+                    line = line.substring(6,line.indexOf("/")) + "(" + time + ")";
+                    duke.addList(line,"E");
+                    break;
```

Comment:
Can consider assigning the magic numbers like 5,6,9 to constant names! Makes the code more easily understandable